.github/workflows/ruby.yml

```diff
@@ -14,12 +14,19 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    # older versions of Ruby want openssl1 which is not on ubuntu-latest
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
         ruby-version:
           - "2.7"
-          - "3.0"
+          #- "3.0"
+        solargraph-version:
```

Comment:
moved to splitting by solargraph version instead of rails version (i.e. we run all rails tests for every [ruby, solargraph] pair in a single job).